autobotAI_integrations/integrations/azure_devops/__init__.py

```
@@ -0,0 +1,305 @@
+            token_url = f"https://login.microsoftonline.com/{self.integration.tenant_id.strip()}/oauth2/v2.0/token"
+            data = {
+                "grant_type": "client_credentials",
+                "client_id": self.integration.client_id.strip(),
+                "client_secret": self.integration.client_secret.strip(),
+                "scope": "499b84ac-1321-427f-aa17-267ca6975798/.default",
+            }
+            resp = requests.post(token_url, data=data, timeout=15)
+            if resp.status_code == 200:
+                token_data = resp.json()
+                return f"Bearer {token_data.get('access_token')}"
+            else:
```

Ruff (0.15.21)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)